bin/thumbnail

```diff
@@ -2,7 +2,7 @@
 # Wrapper for imagemagick that encapsulates our thumbnail logic.
 # @author Damon Snyder (damon@huddler-inc.com) 
 
-IMAGEMAGICK_BASE=${IMAGEMAGICK_BASE:-"/usr/local/"}
```

Comment:
@drsnyder @nmonterroso — the trailing slash seems unnecessary as the variable is only used to define `CONVERT` and `IDENTIFY` below.